content/en/providers/operations-manuals/howto01_using_igtf_ca_distribution/_index.md

```diff
@@ -0,0 +1,322 @@
+centre_. You must deploy the additional trust anchor meta-packages and the new
+local policies in unison, and never install the _cam_ package without such
+software support.
+
+To enable the combined assurance/adequacy model policy, use:
+
+```shell
+yum install ca-policy-egi-cam
+```
+
+or the equivalent for your distribution or operating system. A tarball
+installation is also available.
+
+#### Implementation notes for trust anchor releases post 1.86
+
+Not installing the new "cam" package does not have any detrimental effect on
+current users - only a new class of users (that can only obtain an opaque
+identifiers and do not do full vetting at their electronic identity provider)
+could be affected, and then only those users that are member of one of the
+communities that has part of the combined-assurance programme: LCG-Atlas,
+LCG-Alice, LCG-LHCb, and LCG-CMS.
+
+Also technically this means that you must **only** install the new
+`ca-policy-egi-cam` packages if you **also** at the _same time_ implement
+VO-specific authorization controls in your software stack. This may require
+reconfiguration or a software update:
+
+- for dCache : full support is expected in release v3.1. Meanwhile, if you
+  support enumerated communities for at the DOGWOOD assurance level, you can
+  upgrade to at least release 2.16 and configure negative controls to limit the
+  set of acceptable VOs for those principals whose credential policy OID equals
+  `1.2.840.113612.5.2.2.6`. See [dCache.org](http://www.dCache.org/) for details
+  or ask your software provider. In future releases, differentiated access to
+  storage based on the assurance level can be added - for now decisions are best
+  implemented as binary choices (either 'in' or 'out') for gLExec/LCMAPS : all
+  necessary software is included in UMD-3 and UMD-4, for EL6, EL7, and
+  Debian-based distributions. You can use the example configuration file
+  included in the `ca-policy-egi-cam` package under `/usr/share/doc` and see for
+  more details
+  [the VO-CA-AP Wiki](http://wiki.nikhef.nl/grid/Lcmaps-plugins-vo-ca-ap).
+  Authorization based on VO and CA combinations are binary (either 'in' or
+  'out') for Argus : install at least version 1.7.1 and follow the instructions
+  at
+  [the Argus documentation to enable the new PIP and include new policies in the PAP](http://argus-documentation.readthedocs.io/en/latest/release_notes/v_1_7_1.html)
+
+**When in doubt** just only install the regular `ca-policy-egi-core` package.
+This is what you get anyway when you upgrade using standard tools, and there are
+no changes in this case. The `ca-policy-egi-core` package is approved for all
+VOs membership and assurance models.
+
+## Patches and work-arounds
+
+### mod_ssl renegotiation timeouts
+
+We provide here a workaround for the issue
+[summarised in comment \#57 of bug \#48458](https://savannah.cern.ch/bugs/?func=detailitem&item_id=48458#comment57).
```

Comment:
@dlgroep Savannah is no more available, is this mod_ssl stuff still relevant?

Reply:
I think that's indeed obsolete now...

Reply:
Review comments in-line in the code

Reply:
Thanks, implemented.

Reply:
Dropped.